.github/workflows/build.yml

```diff
@@ -12,28 +17,30 @@ jobs:
   async:
     name: Build using `async` feature
```

Comment:
minor nit: since all our job names start with lower case, we should probably change this to "build using..." and similar comment for "build for non-std".